tests/test_http.py

```diff
@@ -174,10 +174,10 @@ def request(self, *args, **kwargs):
             # set errno to a non-retriable value
             try:
                 # For Windows:
-                ex.errno = socket.errno.WSAECONNREFUSED
+                ex.errno = socket.errno.WSAEHOSTUNREACH
```

Comment:
Since `ECONNREFUSED` is now retried, we need to switch these to a different error that won't be retried